istioctl/cmd/proxyconfig.go

```diff
@@ -852,10 +860,19 @@ func bootstrapConfigCmd() *cobra.Command {
 			if err != nil {
 				return err
 			}
-			return configWriter.PrintBootstrapDump()
+			switch outputFormat {
+			case yamlOutput:
+				return configWriter.PrintBootstrapDump(outputFormat)
+			default:
+				return configWriter.PrintBootstrapDump(jsonOutput)
+				// TODO: cobra default value of global flag is passed instead of local flag
+				// i.e short is passed instead of json as declared below
+				// return fmt.Errorf("output format %q not supported", outputFormat)
```

Comment:
The default condition should always be there to handle unwanted conditions.

Reply:
I wanted to put error message like in other cases. But short is the global default value for proxy-config and that option is not overridden even though child command **bootstrap** is overriding it. So, if no args are passed value **short** will come. So, if it is yaml I am displaying in yaml else in json without checking the value provided.

Reply:
@shamsher31 Is this okay? shall I resolve this?